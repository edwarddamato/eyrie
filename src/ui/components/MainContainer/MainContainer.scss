.main {
  margin: 0 3rem 3rem 3rem;
}