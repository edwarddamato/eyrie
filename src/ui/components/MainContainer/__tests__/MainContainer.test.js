import React from 'react';
import renderer from 'react-test-renderer';
import MainContainer from '../';

describe('MainContainer', () => {
  it('renders correctly', () => {
    const mainContainer = renderer.create(
      <MainContainer><div>foo</div></MainContainer>
    ).toJSON();
    expect(mainContainer).toMatchSnapshot();
  });
});
