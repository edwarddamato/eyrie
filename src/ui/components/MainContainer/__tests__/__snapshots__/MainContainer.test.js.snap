// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer renders correctly 1`] = `
<div
  className="main"
>
  <div>
    foo
  </div>
</div>
`;
